tsl/test/expected/compression_permissions.out

```diff
@@ -136,3 +156,190 @@ REVOKE SELECT on conditions FROM :ROLE_DEFAULT_PERM_USER_2;
+    );
+SELECT table_name FROM create_hypertable( 'conditions', 'timec',
+       chunk_time_interval => '1 week'::interval);
+ table_name 
+------------
+ conditions
+(1 row)
+
+ALTER TABLE conditions SET (
+      timescaledb.compress,
+      timescaledb.compress_segmentby = 'location',
+      timescaledb.compress_orderby = 'timec'
+);
+INSERT INTO conditions
+SELECT generate_series('2018-12-01 00:00'::timestamp, '2018-12-31 00:00'::timestamp, '1 day'), 'POR', 55, 75;
+SELECT compress_chunk(show_chunks('conditions'));
+             compress_chunk              
+-----------------------------------------
+ _timescaledb_internal._hyper_3_6_chunk
+ _timescaledb_internal._hyper_3_7_chunk
+ _timescaledb_internal._hyper_3_8_chunk
+ _timescaledb_internal._hyper_3_9_chunk
+ _timescaledb_internal._hyper_3_10_chunk
+(5 rows)
+
+-- Check that ACL propagates to compressed hypertable.
+\x on
+SELECT * FROM hypertable_details WHERE hypertable = 'public.conditions';
+-[ RECORD 1 ]--+-----------------------------------------------
+hypertable     | public.conditions
+hypertable_acl | 
+compressed     | _timescaledb_internal._compressed_hypertable_4
+compressed_acl | 
+
+SELECT htd.hypertable, htd.hypertable_acl, chunk, chunk_acl
+  FROM chunk_details chd JOIN hypertable_details htd ON chd.hypertable = htd.compressed;
+-[ RECORD 1 ]--+------------------------------------------------
+hypertable     | public.conditions
+hypertable_acl | 
+chunk          | _timescaledb_internal.compress_hyper_4_11_chunk
+chunk_acl      | 
+-[ RECORD 2 ]--+------------------------------------------------
+hypertable     | public.conditions
+hypertable_acl | 
+chunk          | _timescaledb_internal.compress_hyper_4_12_chunk
+chunk_acl      | 
+-[ RECORD 3 ]--+------------------------------------------------
+hypertable     | public.conditions
+hypertable_acl | 
+chunk          | _timescaledb_internal.compress_hyper_4_13_chunk
+chunk_acl      | 
+-[ RECORD 4 ]--+------------------------------------------------
+hypertable     | public.conditions
+hypertable_acl | 
+chunk          | _timescaledb_internal.compress_hyper_4_14_chunk
+chunk_acl      | 
+-[ RECORD 5 ]--+------------------------------------------------
+hypertable     | public.conditions
+hypertable_acl | 
+chunk          | _timescaledb_internal.compress_hyper_4_15_chunk
+chunk_acl      | 
+
+GRANT SELECT ON conditions TO :ROLE_DEFAULT_PERM_USER;
+SELECT * FROM hypertable_details WHERE hypertable = 'public.conditions';
+-[ RECORD 1 ]--+---------------------------------------------------------------
+hypertable     | public.conditions
+hypertable_acl | {super_user=arwdDxt/super_user,default_perm_user=r/super_user}
+compressed     | _timescaledb_internal._compressed_hypertable_4
+compressed_acl | {super_user=arwdDxt/super_user,default_perm_user=r/super_user}
+
+SELECT htd.hypertable, htd.hypertable_acl, chunk, chunk_acl
+  FROM chunk_details chd JOIN hypertable_details htd ON chd.hypertable = htd.compressed;
```

Comment:
needs an ORDER BY chunk so that the order is consistent across runs.

Reply:
Done.